src/stan-users-guide/using-stanc.qmd

```diff
@@ -647,7 +647,9 @@ Warning:
     Left-hand side of distribution statement (~) may contain a non-linear
     transform of a parameter or local variable. If it does, you need
     to include a target += statement with the log absolute determinant
-    of the Jacobian of the transform.
+    of the Jacobian of the transform. You could also consider defining
```

Comment:
You could also consider defining -> Alternatively, you could define
using -> use

Reply:
This is a copy of a warning message stanc3 emits in pedantic mode, so it's wording would need to be changed there

Reply:
Aha.  Our wording's not so great and if we change it, all hell breaks loose on the R side given their goofy unit testing in other packages.